sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -48,59 +48,66 @@ export interface RetryOptions {
+   * If no id is provided, the service will determine the partition to which the event will be sent.
    */
   partitionId?: string;
   /**
    * @property
-   * Retry options for the send operation on the producer. If no value is provided here, the
-   * retry options set when creating the `EventHubClient` is used.
+   * The retry options used to govern retry attempts when an issue is encountered while sending.
+   * If no value is provided here, the retry options set when creating the `EventHubClient` is used.
    */
   retryOptions?: RetryOptions;
 }
 
 /**
- * Options that can be passed when sending a batch of events using the EventHubsClient.
+ * The set of options that can be specified to influence the way in which events are sent to the
+ * Event Hubs service.
  */
 export interface SendOptions {
   /**
    * @property
-   * If specified, Event Hubs will hash this string to map to a partitionId.
+   * The partition hashing key to associate with the event or batch of events.
    * It guarantees that messages with the same partitionKey end up in the same partition.
    * Specifying this will throw an error if the producer was created using a `paritionId`.
    */
   partitionKey?: string | null;
   /**
    * @property
-   * Can cancel the current operation.
+   * An implementation of `AbortSignalLike` to signal the request to cancel the operation.
    */
   abortSignal?: AbortSignalLike;
 }
 
 /**
- * Options that can be passed to the receive operations on the EventHubsClient.
- * @interface EventHubConsumerOptions
+ * The set of options to configure the behavior of an `EventHubConsumer`.
+ * These can be specified when creating the consumer using the `createConsumer` method.
  */
 export interface EventHubConsumerOptions {
   /**
    * @property
-   * The level that this consumer is currently using for partition ownership.
-   * If another consumer is currently active for the same partition with no or lower
-   * level, then it will get disconnected.
-   * If another consumer is currently active with a higher level, then this consumer
-   * will fail to connect.
+   * The owner level associated with an exclusive consumer; for a non-exclusive consumer, this value should be null or undefined.
+   *
+   * When populated, the owner level indicates that a consumer is intended to be the only receiver of events for the
+   * requested partition and an associated consumer group. To do so, this consumer will attempt to assert ownership
+   * over the partition; in the case where more than one exclusive consumer attempts to assert ownership for the same
+   * partition/consumer group pair, the one having a larger `ownerLevel` value will "win."
+   *
+   * When an exclusive consumer is used, other consumers which are non-exclusive or which have a lower owner level will either
+   * not be allowed to be created or if they already exist, will encounter an exception during the next attempted operation.
```

Comment:
I have a few concerns here:
- These are options, we usually don't mention when the value should be null/undefined. We only describe the behavior for when the value is provided
- populated vs provided?
- "not allowed to be created": This statement holds true for the underlying AMQP link, but not for the "consumer" object we give the user. The service can only deny the creation of AMQP link when the link is atttempted to be established which in turn only happens in the first receive operation and not when these options are provided to `createConsumer`
- There are elements in the second and third sections that can be combined.

How about the below?



> When provided, the owner level indicates that a consumer is intended to be the exclusive receiver of events for the
requested partition and the associated consumer group. If other consumers exist for the same
partition - consumer group pair, then the ones with the lower or no `ownerLevel` will get a **add error name here** during
the next attempted receive operation.

Reply:
Remember to update `ownerLevel` in the `EventHubConsumer` class as well